geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreParams.scala

```diff
@@ -20,23 +22,27 @@ object AccumuloDataStoreParams extends GeoMesaDataStoreParams with SecurityParam
     new GeoMesaParam[String](
       "accumulo.instance.id",
       "Accumulo Instance ID",
-      optional = false,
       deprecatedKeys = Seq("instanceId", "accumulo.instanceId"),
       supportsNiFiExpressions = true)
 
   val ZookeepersParam =
     new GeoMesaParam[String](
       "accumulo.zookeepers",
       "Zookeepers",
-      optional = false,
       deprecatedKeys = Seq("zookeepers"),
       supportsNiFiExpressions = true)
 
+  val ZookeeperTimeoutParam =
```

Comment:
This is only used for Accumulo 2.0, right?  

If so, is it worth adding something about that in the description so that Accumulo 1.9 NiFi/GeoServer users don't get excited about tuning this parameter?

Reply:
this will also be used for accumulo 1.9, we just didn't expose it as a ds param before, it was only as a system prop.